BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/ReturnAssistant/LineItemDetailsViewController.swift

```diff
@@ -79,13 +79,6 @@ class LineItemDetailsViewController: UIViewController {
         super.viewWillDisappear(animated)
         guard let lineItem = lineItem, !lineItem.isUserInitiated else { return }
```

Comment:
This line has no use anymore. There method is empty after the guard so there is nothing to guard...

The whole viewWillDisappear method has no use anymore and can be deleted.

Reply:
Good catch 😉